### ISSUE TITLE
CallerlinenumberAttribute bug

### ISSUE THREAD

**Issue by kovlevi** [radimitrov/CSharpShellApp#214]
If I use CallerLinenumberAttribute in method, the returned number is not correct, e.g in one of the source code files the app displays number 57,but it's in number 48,and every call is +9 than the correct number of line in this file.

**Comment by radimitrov** [comment]
Not exactly a bug, just a nececity. Before compiling a source file the app adds some bits of source code like this:
`using Console = CSharpShell.Console;`
Without that you will get loads of PlatformNotImplementedException from System.Console.
I'll try condensing it all with the first code line. That should work.

**Comment by radimitrov** [comment]
I just had to revert the fix for this. It will now always be an "off by one" bug.

**Comment by Alisis33** [comment]
I think related to the # bug #223? 
Maybe add the lines after existing #define's because, even before the bugfix was some issues with #define's. They must be first.

Not tested with current version. Out of memory...
//#define one
#define two

**Comment by radimitrov** [comment]
Yes, it is. And that has always been the solution. I just modified it to only use one line for aliases

**Comment by Alisis33** [comment]
To hafe global aliases would be a nice feature, maybe a featurerequest for the compiler developers?
